mscore/palette.cpp

```diff
@@ -276,7 +282,7 @@ static void applyDrop(Score* score, ScoreView* viewer, Element* target, Element*
       DropData dropData;
       dropData.view       = viewer;
       dropData.pos        = pt;
-      dropData.dragOffset = pt;
+      dropData.dragOffset = QPointF();
```

Comment:
It seems that either way, the one case where it seems relevant doesn't work right: adding a symbol to another symbol via double click.  The problem is that *neither* pos nor dragOffset will have meaningful values in that case - they are both set to 0.  Seems we really need to set pt the same way here as in the code below for measures - we need the beginning-of-score-relative position of the target, since that is what drop() relies on.  But it's not directly relevant to my changes here.